src/System.Runtime.Extensions/tests/VoidMainWithExitCodeApp/VoidMainWithExitCodeApp.csproj

```diff
@@ -4,11 +4,9 @@
   <PropertyGroup>
     <ProjectGuid>{9F312D76-9AF1-4E90-B3B0-815A1EC6C346}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <RootNamespace>VoidMainWithExitCodeApp</RootNamespace>
-    <AssemblyName>VoidMainWithExitCodeApp</AssemblyName>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <CopyNuGetImplementations>false</CopyNuGetImplementations>
-    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.3</NuGetTargetMoniker>
+    <IsTestProject>true</IsTestProject>
+    <NoWarn>0436</NoWarn>
```

Comment:
This is actually because of the `ExcludeFromCodeCoverageAttribute`. Here's what I wrote in the earlier version of this PR:

> This appears in some of the test projects because they are using the ExcludeFromCodeCoverageAttribute and applying it to the whole assembly. We are exposing that attribute publically now, but the public version is not allowed to be applied to assemblies. So we're still using a duplicate locally-compiled type and just suppressing the ambiguity warning.